pvactools/lib/aggregate_all_epitopes.py

```diff
@@ -452,6 +455,17 @@ def get_tier(self, mutation, vaf_clonal):
             if mutation["Transcript Support Level"] > self.maximum_transcript_support_level:
                 tsl_pass = False
 
+        ic50_pass = mutation["{} MT IC50 Score".format(self.mt_top_score_metric)] < binding_threshold
+        percentile_pass = (
+            self.percentile_threshold is not None and 
```

Comment:
I believe the current logic will set `percentile_pass` to `False` if `self.percentile_threshold` is not set. But it should be set to `True` in that case.